osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -268,6 +268,11 @@ public virtual void Save(BeatmapInfo info, IBeatmap beatmapContent, ISkin beatma
 
                     // metadata may have changed; update the path with the standard format.
                     beatmapInfo.Path = $"{metadata.Artist} - {metadata.Title} ({metadata.Author}) [{beatmapInfo.Version}].osu";
+
+                    // remove incompatible characters from path
+                    foreach (char c in Path.GetInvalidFileNameChars())
```

Comment:
This already exists in `ArchiveModelManager`. Please use that function instead.

Reply:
If you're referring to this one, it's a private function that's why I couldn't use it. What do you suggest?
https://github.com/ppy/osu/blob/f5a5b87cf6e633e237b7595b1ef9cbf3dffe3da7/osu.Game/Database/ArchiveModelManager.cs#L896-L901

Reply:
make it not private. in this case `protected static` would likely work fine.

Reply:
Done

Reply:
Please don't self-resolve reviews. We will resolve them once we have re-checked the issue. Also there's no need to reply with "done" - github alerts reviewers with notifications when a change is pushed.